java/ql/test/library-tests/dataflow/taintsources/IntentSources.java

```diff
@@ -27,6 +27,7 @@ public void test3() throws java.io.IOException {
 
 }
 
+
```

Comment:
Accidental change?

Reply:
Yes, it was the auto-formatter deciding to complicate my life :-P 

Fixed in 22b99f1, thanks!